x-pack/packages/kbn-elastic-assistant/impl/data_anonymization/settings/anonymization_settings_management/index.tsx

```diff
@@ -6,70 +6,127 @@
-  const onListUpdated = useAnonymizationListUpdate({
-    anonymizationFields,
+const AnonymizationSettingsManagementComponent: React.FC<Props> = ({ defaultPageSize = 5 }) => {
+  const { toasts } = useAssistantContext();
+  const { data: anonymizationFields } = useFetchAnonymizationFields();
+  const [hasPendingChanges, setHasPendingChanges] = useState(false);
+
+  const {
     anonymizationFieldsBulkActions,
     setAnonymizationFieldsBulkActions,
     setUpdatedAnonymizationData,
+    resetSettings,
+    saveSettings,
+    updatedAnonymizationData,
+  } = useSettingsUpdater(
+    DEFAULT_CONVERSATIONS, // Anonymization settings do not require conversations
+    DEFAULT_PROMPTS, // Anonymization settings do not require prompts
+    false, // Anonymization settings do not require conversations
+    anonymizationFields ?? DEFAULT_ANONYMIZATION_FIELDS
+  );
+
+  const onCancelClick = useCallback(() => {
+    resetSettings();
+    setHasPendingChanges(false);
+  }, [resetSettings]);
+
+  const handleSave = useCallback(
+    async (param?: { callback?: () => void }) => {
+      await saveSettings();
+      toasts?.addSuccess({
+        iconType: 'check',
+        title: SETTINGS_UPDATED_TOAST_TITLE,
+      });
+      setHasPendingChanges(false);
+      param?.callback?.();
+    },
+    [saveSettings, toasts]
+  );
+
+  const onSaveButtonClicked = useCallback(() => {
+    handleSave();
+  }, [handleSave]);
+
+  const handleAnonymizationFieldsBulkActions = useCallback(
+    (value) => {
+      setHasPendingChanges(true);
+      setAnonymizationFieldsBulkActions(value);
+    },
+    [setAnonymizationFieldsBulkActions]
+  );
+
+  const handleUpdatedAnonymizationData = useCallback(
+    (value) => {
+      setHasPendingChanges(true);
+      setUpdatedAnonymizationData(value);
+    },
+    [setUpdatedAnonymizationData]
+  );
+
+  const onListUpdated = useAnonymizationListUpdate({
+    anonymizationFields: updatedAnonymizationData,
+    anonymizationFieldsBulkActions,
+    setAnonymizationFieldsBulkActions: handleAnonymizationFieldsBulkActions,
+    setUpdatedAnonymizationData: handleUpdatedAnonymizationData,
   });
   return (
-    <EuiPanel hasShadow={false} hasBorder paddingSize="l">
-      <EuiTitle size={'xs'}>
-        <h2>{i18n.SETTINGS_TITLE}</h2>
-      </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiText size={'xs'}>{i18n.SETTINGS_DESCRIPTION}</EuiText>
-
-      <EuiSpacer size="m" />
-
-      <EuiFlexGroup alignItems="center" data-test-subj="summary" gutterSize="none">
-        <Stats
-          isDataAnonymizable={true}
-          anonymizationFields={anonymizationFields.data}
-          titleSize="m"
-          gap={euiThemeVars.euiSizeS}
-        />
-      </EuiFlexGroup>
+    <>
+      <EuiPanel hasShadow={false} hasBorder paddingSize="l">
+        <EuiTitle size={'xs'}>
+          <h2>{i18n.SETTINGS_TITLE}</h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiText size={'xs'}>{i18n.SETTINGS_DESCRIPTION}</EuiText>
```

Comment:
For consistency I think we can get rid of the title here and make the text size the same as on the other tabs, that okay @bojanasan?

Anonymization:
<img width="1566" alt="image" src="https://github.com/elastic/kibana/assets/2946766/125b54fb-c831-4fb1-a006-5bc62cd70494">
Quick Prompts:
<img width="1569" alt="image" src="https://github.com/elastic/kibana/assets/2946766/cb48a856-4f70-4b43-aac9-7efc4ceceafd">
System Prompts:
<img width="1570" alt="image" src="https://github.com/elastic/kibana/assets/2946766/e736aa95-3c47-465b-addb-5e47d059eb88">

Reply:
@spong Agreed, lets do that. I've updated the mocks to the correct and consistent sizing.